optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -432,6 +439,9 @@ def __init__(
         if valid_sets is None:
             raise ValueError("`valid_sets` is required.")
 
+        self.optuna_callbacks = optuna_callbacks
+        self.start_time = None  # type: Optional[float]
```

Comment:
[IMO] How about removing `self.start_time`? This is un-documented and not used.

Reply:
Agree. I'll remove `start_time` in this PR.

Reply:
Thanks!